pygeos/test/test_io.py

```diff
@@ -248,6 +245,51 @@ def test_to_wkb_srid():
     assert np.frombuffer(result[5:9], "<u4").item() == 4326
 
 
+def test_to_wkb_point_empty():
+    # empty point converts to POINT (nan, nan, nan) in WKB
+    # this matches PostGIS behaviour
+    assert pygeos.to_wkb(empty_point) == POINT_NAN_WKB
+
+
+def test_from_wkb_point_nan():
+    # WKB representation of POINT (nan, nan, nan) converts to POINT EMPTY
+    geom = pygeos.from_wkb(POINT_NAN_WKB)
```

Comment:
In the reading case, should we test that both variations of the POINT_NAN_WKB (with 2 or 3 coordinates) are read correctly? (because now this test only tests one of both depending on the GEOS version)